### PR TITLE
fix: remove scrollbar produced when QR button is clicked

### PR DESCRIPTION
**Pull request by Kingsuite**

### CODE REVIEW

File: packages/nextjs/styles/globals.css [hunk 1]
```diff
@@ -27,9 +27,28 @@ p {
   @apply shadow-md;
 }
 
+html {
+  @apply overflow-y-scroll;
+}
+
+body {
+  @apply overflow-y-auto;
+}
```
Comment (metalboyrick) [inline packages/nextjs/styles/globals.css:36]: can consider hiding overflow x? what do you think?

**Review by metalboyrick — CHANGES_REQUESTED** [review]
minor comments!

File: packages/nextjs/styles/globals.css (hunk 1 above)
Comment (Nadai2010) [reply packages/nextjs/styles/globals.css:36]: Hi @Kingsuite please remember the required changes

Comment (Kingsuite) [reply packages/nextjs/styles/globals.css]: GM @Nadai2010 @metalboyrick, sorry for the delay. Changes have been implemented. Thanks for your review

**Pull request merged**